src/main/java/sawkiewicz/task1/Item.java

```diff
@@ -0,0 +1,61 @@
+package sawkiewicz.task1;
+
+public class Item {
+    private char name;
+    private int unit;
+
+    public Item(char name, int unit) {
+        this.name = name;
+        this.unit = unit;
+    }
+
+    public char getName() {
+        return name;
+    }
+
+    public void setName(char name) {
+        this.name = name;
+    }
+
+    public int getUnit() {
+        return unit;
+    }
+
+    public void setUnit(int unit) {
+        this.unit = unit;
+    }
+
+    public int giveTotalPrice() {
+        int price = 0;
+        int specialUnitNumber = 0;
+        int specialPrice = 0;
+
+        switch (name) {
```

Comment:
you should pull products from repository, please read

https://medium.com/@krzychukosobudzki/repository-design-pattern-bc490b256006